pom.xml

```diff
@@ -17,15 +17,15 @@
 
   <properties>
     <project.build.resourceEncoding>${project.build.sourceEncoding}</project.build.resourceEncoding>
-    <version.java>1.8</version.java>
+    <version.java>11</version.java>
     <java.version>${version.java}</java.version>
 
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
 
 
-    <zeebe.version>0.20.0</zeebe.version>
-    <spring-boot.version>2.1.4.RELEASE</spring-boot.version>
+    <zeebe.version>0.21.1</zeebe.version>
+    <spring-boot.version>2.1.7.RELEASE</spring-boot.version>
```

Comment:
@spulci can we update to 2.2.0.RELEASE instead?

Reply:
Compile and test passed locally elevating to version 2.2.0.RELEASE